app/serializers/basic_post_serializer.rb

```diff
@@ -36,21 +35,11 @@ def cooked
       else
         I18n.t('flagging.user_must_edit')
       end
-    elsif ignored
-      I18n.t('ignored.hidden_content')
     else
       object.filter_quotes(@parent_post)
     end
   end
 
-  def ignored
```

Comment:
Was there any tests for this that needs to be removed?

Reply:
No, I wrote tests in another PR, but that is already closed.